docs/sdk/src/polkadot_sdk/polkadot.rs

```diff
@@ -83,5 +89,28 @@
 //!   Agile periodic-sale-based model for assigning Coretime on the Polkadot Ubiquitous Computer.
 //! - RFC#5: [Coretime-interface](https://github.com/polkadot-fellows/RFCs/blob/main/text/0005-coretime-interface.md):
 //!   Interface for manipulating the usage of cores on the Polkadot Ubiquitous Computer.
-// TODO: add more context and explanations about Polkadot as the Ubiquitous Computer and related
-// tech. <https://github.com/paritytech/polkadot-sdk-docs/issues/66>
+//!
+//! ### Polkadot, a Ubiquitous Computer
+//!
+//! Polkadot can be described as a "ubiquitous computer" based on its architecture. It is evolving
+//! beyond its initial concept into a platform focusing on blockspace (the fundamental resource
+//! chains need) rather than the chains themselves. Polkadot serves as a general-purpose provider
+//! for resilient computation, offering a multicore computer-like environment where various
+//! parachains operate in parallel on different cores. This approach allows applications to access
+//! secure and decentralized computation flexibly and efficiently, marking a shift from a
+//! chain-centric to an application-centric ecosystem.
+//!
+//! #### We can visualize it as follows
+//! [![Polkadot as the Ubiquitous Computer](https://pbs.twimg.com/media/F3qCltlWkAECsWJ?format=jpg&name=medium)](https://twitter.com/filippoweb3/status/1692025249070563413)
+//! Let's break down the "space" layers from the above diagram:
+//! - **Application Space**: This is the space where each parachain's application logic lives as an
+//!   independent, but interconnected, blockchain.
+//! - **Accord Space**: Is the space for opt-in treaties across chains. Treaty's members cannot
+//!   change/undermine the treaty's logic. This allows trustless cross-chain applications.
+//! - **Core Space-time**: This can be seen as the CPU of the Polkadot Ubiquitous Computer, where
+//!   the processes are queued and executed. In oder words, we can say that the *Core Space-time*
+//!   "consumes" *Coretime* using Polkadot computation. Here is where collators send blocks through
+//!   cores to validators for validity and availability checks.
+//!
+//! Additionally, this introduces the concept of **Coretime**. Coretime is the right to use
+//! Polkadot's computing power. Sold through a broker system parachain.
```

Comment:
🤮  Sorry but now this is just throwing around a bunch of terms and makes it more confusing, not more clear.